app/assets/javascripts/discourse/tests/acceptance/keyboard-shortcuts-test.js

```diff
@@ -41,3 +42,67 @@ acceptance("Keyboard Shortcuts", function (needs) {
     assert.equal(currentURL(), "/t/keyboard-shortcuts-are-awesome/27331");
   });
 });
+
+acceptance("Keyboard Shortcuts - Authenticated Users", function (needs) {
+  let resetNewCalled = 0;
+  let markReadCalled = 0;
+  needs.user();
+  needs.pretender((server, helper) => {
+    let topicList = DiscoveryFixtures["/latest.json"];
+
+    // get rid of some of the topics and the more_topics_url
+    // so we consider them allLoaded and show the footer with
+    // the bottom dismiss button
+    topicList.topic_list.topics.splice(20, 30);
+    topicList.topic_list.more_topics_url = null;
```

Comment:
Looks good, but I'm wondering if the fixture mutated is mutated for this test case only or is it carried over to the tests that run after this one?

Reply:
Hmmm that is a very good point, let me run it with another test using the same `DiscoveryFixture` and see

Reply:
You were right @OsamaSayegh this modified state, changed to clone the fixture first before modifying it